inventory-provisioning-web/src/main/scala/com/normation/inventory/provisioning/endpoint/config/AppConfig.scala

```diff
@@ -93,6 +96,15 @@ class AppConfig {
   @Value("${waiting.inventory.queue.size}")
   var WAITING_QUEUE_SIZE = 50
 
+  @Value("${inventories.root.directory}")
+  var INVENTORY_ROOT_DIR = ""
```

Comment:
Should'nt we set /var/rudder/inventories as default ?

Reply:
Because it's spring, it's not a real default (ie it will fail if it did not find the config prop). So for real, it's just an info for the developpers.